package.json

```
@@ -138,6 +138,11 @@
                     "default": 90000,
                     "markdownDescription": "%AWS.configuration.description.samcli.lambdaTimeout%"
                 },
+                "aws.samcli.enableLegacyDeploy": {
```

nit: usually don't need "enable" in settings names, it is implied
```suggestion
                "aws.samcli.legacyDeploy": {
```